packages/block-library/src/comments-pagination-previous/edit.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { useBlockProps, PlainText } from '@wordpress/block-editor';
+
+const arrowMap = {
+	none: '',
+	arrow: '←',
+	chevron: '«',
+};
+
+export default function CommentsPaginationPreviousEdit( {
+	attributes: { label },
+	setAttributes,
+	context: { paginationArrow },
+} ) {
+	const displayArrow = arrowMap[ paginationArrow ] || arrowMap.none;
+	return (
+		<div { ...useBlockProps() }>
+			<a
+				href="#comments-pagination-previous-pseudo-link"
+				onClick={ ( event ) => event.preventDefault() }
+			>
+				<PlainText
+					__experimentalVersion={ 2 }
+					tagName="span"
+					aria-label={ __( 'Previous comments page link' ) }
+					placeholder={ __( 'Previous Comments' ) }
+					value={ label }
+					onChange={ ( newLabel ) =>
+						setAttributes( { label: newLabel } )
+					}
+				/>
+				{ displayArrow && (
+					<span
+						className={ `wp-block-query-pagination-previous-arrow` }
```

Comment:
It looks like a copy and paste issue. Should it be the following?

```suggestion
						className=“wp-block-comments-pagination-previous-arrow”
```